gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Aug 30 12:45:41 PDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Looks like Jar/wrapper changes are missing. Run:
```
./gradlew wrapper --gradle-version 5.6.1 --distribution-type all; ./gradlew wrapper --gradle-version 5.6.1 --distribution-type all
```

Reply:
You ever wonder what we'd have room for in our heads if they weren't filled with gradle commands?